### PR TITLE
Add Nuget.exe as Prerequisites

### PR DESCRIPTION
**Pull request by jahbenjah**
You need the nuget.exe to create a custom template

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Comment by mairaw** [comment]
@KathleenDollard @dsplaisted @guardrex can you confirm that prereq for creating a custom template?

**Comment by guardrex** [comment]
I don't think [dotnet pack](https://docs.microsoft.com/dotnet/core/tools/dotnet-pack) existed at the time we put this topic out. If `dotnet pack` hits the spot, then I'd resolve this by swapping the `dotnet pack` command into the topic.

**Comment by guardrex** [comment]
mmmmm 🤔 .... *existed* ... let me rephrase to "*worked*".

> Currently, a custom template is packed on Windows with [nuget.exe](https://dist.nuget.org/win-x86-commandline/latest/nuget.exe) (not [dotnet pack](dotnet-pack.md)).

... **_"not dotnet pack"_** ... 

For example at the time ... https://github.com/dotnet/dotnet-template-samples/issues/16#issuecomment-310690785

The explicit "not dotnet pack" comment probably reflects a limitation of the `dotnet pack` command at the time.

I'm free next weekend. Would u like me to run it with `dotnet pack` and see what happens? :smile: or :cry:

**Comment by guardrex** [comment]
@mairaw If you like, I'll be free over the 🎄 holiday break ⛄️ to work it or immediately after my contract ends on 12/31. Let me know.

**Comment by BillWagner** [comment]
@KathleenDollard @rlander Who should look at this for the correct xplat workflow?

**Comment by seancpeters** [comment]
Using dotnet pack should create the nupkg whether it's run on Windows or on Linux. Nupkgs themselves are platform agnostic, and so it should not matter on which platform the nupkg is created.

**Comment by dsplaisted** [comment]
@seancpeters `dotnet pack` creates a .nupkg from a project file.  In this guide for creating templates, you need to create a .nupkg from a .nuspec, which isn't supported by `dotnet pack`, at least not without a special project file.

File: docs/core/tutorials/create-custom-template.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ If you wish to use the downloaded sample with file system distribution, do the f
 ## Prerequisites
 
 - Install the [.NET Core 2.0 SDK](https://www.microsoft.com/net/core) or later versions.
+- Install the [nuget.exe CLI](https://docs.microsoft.com/en-us/nuget/install-nuget-client-tools#nugetexe-cli)
```
Comment (Thraka) [inline docs/core/tutorials/create-custom-template.md]: The link shouldn't have `en-us` in it.

```suggestion
- Install the [nuget.exe CLI](https://docs.microsoft.com/nuget/install-nuget-client-tools#nugetexe-cli)
```

**Review by Thraka — CHANGES_REQUESTED** [review]
En-us should be removed from the URL.

**Comment by Thraka** [comment]
The article is written for the .nuspec file. If that file isn't supported with `dotnet pack` and we need to showcase that scenario, the article needs to be rewritten. This PR though is valid as, according to the article, nuget.exe is used and it's not listed as a prerequisite. According to the nuget install article, you can use nuget.exe on linux by installing mono.

I think we should accept the PR. If we want the article rewritten with the faux csproj that replaces the nuspec and use dotnet pack, we should file a new issue for that.

**Comment by mairaw** [comment]
@Thraka you completely rewrote these tutorials, no? should we close this?

**Comment by Thraka** [comment]
Yes this should be closed. This file doesn't exist anymore.
